flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/fs/hdfs/HadoopFileSystem.java

```diff
@@ -224,8 +224,8 @@ public RecoverableWriter createRecoverableWriter() throws IOException {
 	static FileSystemKind getKindForScheme(String scheme) {
 		scheme = scheme.toLowerCase(Locale.US);
 
-		if (scheme.startsWith("s3") || scheme.startsWith("emr")) {
-			// the Amazon S3 storage
+		if (scheme.startsWith("s3") || scheme.startsWith("emr") || scheme.startsWith("oss")) {
```

Comment:
Maybe this change should go into s separate commit, either with a different jira or as hotfix.

Reply:
Sounds good, I think hotfix will be better.